chevah_build

```diff
@@ -406,6 +406,19 @@ case $OS in
         export CC="clang"
         export CXX="clang++"
         export BUILD_LIBFFI="yes"
+        if [ "$OS" = "openbsd65" ]; then
```

Comment:
this is ok, but I think that we should have support for single openBSD... and that should match the one used on your side.

So, this PR should also remove support for openbsd64

Reply:
I agree with having a single release supported and tested for some OS'es, but I always keep in mind that you never know who is going to try this public repo. 

So, if easily achievable, I try to not break older versions, especially if still supported upstream.